tests/test_vector_sample.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+import unittest
+import time
+from pprint import pprint
+from flask.json import loads as json_load
+from flask.json import dumps as json_dump
+try:
+    from .test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
+except:
+    from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
+
+
+__license__ = "GPLv3"
+__author__ = "Sören Gebbert, Markus Neteler"
+__copyright__ = "Copyright 2016-present, Sören Gebbert and mundialis GmbH & Co. KG"
+
+JSON = {
+    "type": "FeatureCollection",
+    "crs": {"type": "name", "properties": {"name": "urn:ogc:def:crs:OGC:1.3:CRS84"}},
+    "features": [
+        {"type": "Feature", "properties": {"cat": 1},
+         "geometry": {"type": "Point", "coordinates": [638684.0, 220210.0]}},
+        {"type": "Feature", "properties": {"cat": 2},
+         "geometry": {"type": "Point", "coordinates": [635676.0, 226371.0]}}
+    ]
+}
+
+
+class RasterTestCase(ActiniaResourceTestCaseBase):
+
+    # ################### Raster SAMPLING ###################################
+
+    def test_async_sampling(self):
+
+        rv = self.server.post(URL_PREFIX + '/locations/nc_spm_08/mapsets/PERMANENT/vector_layers/zipcodes_wake/'
+                              'sampling_async',
+                              headers=self.user_auth_header,
+                              data=json_dump({"points":[["a", "638684.0", "220210.0"],
+                                                         ["b", "635676.0", "226371.0"]]}),
+                              content_type="application/json")
+
+        pprint(json_load(rv.data))
```

Comment:
Can `pprint` be removed? A debug left-over?

Reply:
yes it can be removed

Reply:
It is part of all tests in that directory. If to be removed, I suggest it should happen in a separate PR.

Reply:
I don't mind when this is removed. Perfectly fine in a follow up.

Reply:
> Can `pprint` be removed? A debug left-over?

Moved to the new issue #10